docs/development/testguide.rst

```diff
@@ -634,7 +630,7 @@ bundled version using e.g.::
 
     from ...tests.helper import pytest
 
-but this is no longer the recommended method.
+but this is no longer supported.
```

Comment:
I'd just remove this whole section about things prior to 2.0 - that's too long ago!

Reply:
Done